packages/api/internal/orchestrator/metrics.go

```diff
@@ -53,8 +53,12 @@ func (o *Orchestrator) setupMetrics(meterProvider metric.MeterProvider) (metric.
 			return nil
 		}, gauge)
 	if err != nil {
-		return nil, fmt.Errorf("failed to register orchestrators gauge: %w", err)
+		return fmt.Errorf("failed to register orchestrators gauge: %w", err)
 	}
 
-	return registration, nil
+	if o.createdSandboxesCounter, err = meter.Int64Counter("api.orchestrator.created_sandboxes"); err != nil {
```

Comment:
can you move the meter to `packages/shared/pkg/telemetry/meters.go`, we have other meters defined there?